app/uk/gov/hmrc/individualsincomeapi/controllers/v1/CommonController.scala

```diff
@@ -89,13 +89,8 @@ abstract class CommonController @Inject()(cc: ControllerComponents) extends Back
       auditHelper.auditApiFailure(correlationId, matchId, request, url, e.getMessage)
       ErrorInvalidRequest(e.getMessage).toHttpResponse
     }
-    case e: InternalServerException => {
-      logger.warn("Controllers InternalServerException encountered")
-      auditHelper.auditApiFailure(correlationId, matchId, request, url, e.getMessage)
-      ErrorInternalServer("Something went wrong.").toHttpResponse
-    }
     case e: Exception => {
-      logger.warn("Controllers Exception encountered")
+      logger.error("Controllers Exception encountered", e)
       auditHelper.auditApiFailure(correlationId, matchId, request, url, e.getMessage)
```

Comment:
Do we still want to audit the exception itself? Doesn't seem like the right thing to do

Reply:
The "audit" is just about sending it to Splunk as well (with extra detail) (so that when we need to look at historical data, we can).